go.mod

```diff
@@ -0,0 +1,17 @@
+// Generated by gohack; DO NOT EDIT.
+module github.com/divisionone/go-micro
+
+go 1.14
+
+require (
+	github.com/divisionone/cli v0.2.1
+	github.com/divisionone/go-rcache v0.0.0-20180409064418-3bf8d45e6c93
+	github.com/divisionone/util v0.0.0-20180409064546-d45e66572862
```

Comment:
After this is merged, we can tag `go-micro`, then update `util` & tag it and come back to update this version spec. Likewise for `go-rcache`.